README.md

```diff
@@ -59,6 +59,7 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.3/install.html#
 | `bbb_mute_on_start:` | start with muted mic on join | `no` | |
 | `bbb_app_log_level:` | set bigbluebutton log level | `DEBUG` | |
 | `bbb_meteor:` | overwrite settings in meteor | `{}` | |
+| `bbb_recording_config:` | overwrite recording settings | `{}` | See [Enable playback of recordings on iOS](https://docs.bigbluebutton.org/admin/customize.html#enable-playback-of-recordings-on-ios). It works like `bbb_meteor` by merging your custom config with the server config. |
```

Comment:
should the value name contain the colon? Looks like other values also include it.

Reply:
Hey @EdJoPaTo, do you mean adding it to `bbb_meteor` in the comment column?

Reply:
```suggestion
| `bbb_recording_config` | overwrite recording settings | `{}` | See [Enable playback of recordings on iOS](https://docs.bigbluebutton.org/admin/customize.html#enable-playback-of-recordings-on-ios). It works like `bbb_meteor` by merging your custom config with the server config. |
```

A bunch of names end with the colon. Not only the one you added here. I wondered If that should be removed. But as its not only in your line that would be a change independent from this PR.

Reply:
Gotcha. I'll do a separate PR to fix the other rows.